### PR TITLE
osd/ReplicatedPG: fix code indentation

### PR DESCRIPTION
**Pull request by runsisi**
misc: fix minor code indentation issue

Signed-off-by: runsisi <runsisi@zte.com.cn>

### CODE REVIEW

File: src/osd/ReplicatedPG.cc [hunk 1]
```diff
@@ -9334,11 +9334,11 @@ SnapSetContext *ReplicatedPG::get_snapset_context(
       if (!(oid.is_head() && !oid_existed))
 	r = pgbackend->objects_get_attr(oid.get_head(), SS_ATTR, &bv);
       if (r < 0) {
-	// try _snapset
-      if (!(oid.is_snapdir() && !oid_existed))
-	r = pgbackend->objects_get_attr(oid.get_snapdir(), SS_ATTR, &bv);
-      if (r < 0 && !can_create)
-	return NULL;
+        // try _snapset
```
Comment (tchaikov) [inline src/osd/ReplicatedPG.cc]: see https://github.com/ceph/ceph/blob/master/CodingStyle#L75

**Comment by tchaikov** [comment]
i am closing as we have tabstop=8.

**Comment by runsisi** [comment]
but what i want to fix most is the lines start from https://github.com/ceph/ceph/blob/master/src/osd/ReplicatedPG.cc#L9338

**Comment by tchaikov** [comment]
@runsisi sorry, gotcha!

**Comment by runsisi** [comment]
@tchaikov never mind, thank u:)

**Comment by yuyuyu101** [comment]
I don't encourage code style fix since we have too much inconsistence files. And even now we still merge different code style commits. Coding style commit will hurt for commit backport and diff for trace problem.

**Comment by theanalyst** [comment]
Agree with @yuyuyu101 here, pure indentation fixes make backporting difficult and also when you trace an issue with git blame etc. you've sort of lost history because of a ws change.. while it is ok when you fix indentation when your patch also touches that code, otherwise a pure ws fix will not buy us  much.

**Comment by runsisi** [comment]
thank you guys for elaborating on this for me :)
i am closing it.

**Pull request closed without merging**